test/e2e/run-all.js

```diff
@@ -84,6 +84,12 @@ async function main() {
             default: false,
             description: 'Update E2E snapshots',
             type: 'boolean',
+          })
+          .option('update-privacy-snapshot', {
+            default: false,
```

Comment:
Once we set up code owners for the snapshot file, as you mentioned, maybe we can make this flag opt-out instead.

Reply:
I feel like the extra friction is a good thing because it draws attention to it. Snapshot updating is always opt-in in all test runners.